blog/2023-year-in-review.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 2023 at CCV
+author: Ellen Duong
+people:
+    - Ashok Ragavendran
+    - August Guang
+    - Eric Salomaki
+    - Jordan Lawson
+    - Joselynn Wallace
+    - Paul Cao
+    - Paul Xu
+    - Anna Murphy
+    - Ashley S. Lee
+    - Bradford N. Roarr
+    - Camilio Diaz
+    - Carlos Paniagua
+    - Ellen Duong
+    - Ford McDonald
+    - Galen Winsor
+    - George Dang
+    - Heather Yu
+    - Isabel Restrepo
+    - John Gerrard Holland
+    - Joshua Lu
+    - Robert Gemma, Jr.
+    - Timothy Divoll
+    - Paul Hall
+    - Pratibha Raghunandan
+    - Pritivi Thakur
+    - David D Johnson
+    - Geoffrey Avila
+    - Nathan Wood
+    - Paulo S Baptista
+    - Singh Saluja
+    - Prasad Bandarkar
+    - Rain Fan
+    - Yang Liu
+    - Jarrod Winsor
+    - Linnea Wolfe
+    - Michael Antonelli
+    - Paul Koussa
+    - Paul Stey
+    - Sam Fulcomer
+    - Thomas Serre
+    - Timothy Poli
+date: 2024-01-11
+slug: 2023-year-in-review
+team: GSDC (Graphics, Software and Data Core)
+project:
+description: 2023 was a year of numerous strides at CCV and in the research community here at Brown. Here are some of the many highlights over the past year!
+tags:
+    - software
+
+---
+ 
+2023 was a year of numerous strides at CCV and in the research community here at Brown. Thank you for your continued participation and support. 
+
+Here are some quick statistics from our Brown community made in 2023. We had 1557 active Oscar users, which is a 24.66% increase in activity from the year prior. Over 3.6 million jobs were run in Oscar totalling over 57 million CPU core hours! Since 2018, the number of active oscar users has increased and well as demand for resources. Over **470** questions were asked and helped in our weekly office hours.
```

Comment:
```suggestion
Here are some quick statistics from our Brown community made in 2023. We had 1557 active Oscar users, which is a 24.66% increase in activity from the year prior. Over 3.6 million jobs were run in Oscar totalling over 57 million CPU core hours! Since 2018, the number of active oscar users has increased as well as demand for resources. Over **470** questions were asked and helped in our weekly office hours.
```